service/controller/aws/key/key.go

```diff
@@ -3,53 +3,10 @@ package key
 import (
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/giantswarm/cluster-operator/service/controller/key"
 )
 
-// AppSpecs returns apps installed only for AWS.
-func AppSpecs() []key.AppSpec {
-	// Add any provider specific charts here.
-	return []key.AppSpec{
-		{
-			App:             "cert-manager",
-			Catalog:         "default",
-			Chart:           "cert-manager-app",
-			ClusterAPIOnly:  true,
-			Namespace:       metav1.NamespaceSystem,
-			UseUpgradeForce: true,
-			Version:         "1.0.4",
-		},
-		{
-			App:             "cluster-autoscaler",
-			Catalog:         "default",
-			Chart:           "cluster-autoscaler-app",
-			Namespace:       metav1.NamespaceSystem,
-			UseUpgradeForce: true,
-			Version:         "1.1.3",
-		},
-		{
-			App:             "external-dns",
-			Catalog:         "default",
-			Chart:           "external-dns-app",
-			ClusterAPIOnly:  true,
-			Namespace:       metav1.NamespaceSystem,
-			UseUpgradeForce: true,
-			Version:         "1.1.0",
-		},
-		{
-			App:             "kiam",
-			Catalog:         "default",
-			Chart:           "kiam-app",
-			ClusterAPIOnly:  true,
-			Namespace:       metav1.NamespaceSystem,
-			UseUpgradeForce: true,
-			Version:         "1.0.3",
-		},
-	}
-}
-
 // ChartSpecs returns charts installed only for AWS.
 func ChartSpecs() []key.ChartSpec {
```

Comment:
This is probably unrelated but why do we still need `ChartSpecs`?

Reply:
It's still used in `newChartSpecsToMigrate`. But since we have had 3+ releases from app migration, it's a safe assumption to delete this `appmigration` for legacy cluster, but not in this PR. @rossf7 WDYT?

Reply:
> But since we have had 3+ releases from app migration, it's a safe assumption to delete this appmigration for legacy cluster, but not in this PR

No sadly that won't work.

Customers can upgrade from old versions like 8.2.0. So the migration logic is needed until there are no chartconfig CRs in any tenant clusters.